conf/webpack-devserver.conf.js

```diff
@@ -0,0 +1,65 @@
+const { resolve } = require('path');
+const webpack = require('webpack');
+
+module.exports = {
+    devtool: 'inline-source-map',
+    context: resolve(__dirname, '..', 'public'),
+    entry: [
+      'react-hot-loader/patch',
+      'webpack/hot/only-dev-server',
+      './js/app.js'
+    ],
+    output: {
+      filename: 'app.js',
+      path: resolve(__dirname, '..', 'public', 'build'),
+      publicPath: 'https://atomworkshop-assets.local.dev-gutools.co.uk/assets/build/'
+    },
+
+    devServer: {
+      hot: true,
+      contentBase: resolve(__dirname, '..', 'public', 'build'),
+      publicPath: 'https://atomworkshop-assets.local.dev-gutools.co.uk/assets/build/',
+      port: 9051,
+      public: "atomworkshop-assets.local.dev-gutools.co.uk"
```

Comment:
I've no objection to doing it for DRY reasons... but I actually think it's clearer it all being written out. Compared to the same file in media atom maker https://github.com/guardian/media-atom-maker/blob/master/build_config/webpack.devserver.conf.js